src/Bicep.Cli/Program.cs

```diff
@@ -130,6 +130,7 @@ private ServiceProvider ConfigureServices()
                 .AddSingleton<DecompilationWriter>()
                 .AddSingleton<CompilationWriter>()
                 .AddSingleton<ParametersFileWriter>()
+                .AddSingleton<ParamsFileWriter>()
```

Comment:
The distinction between `ParametersFileWriter` and `ParamsFileWriter` is confusing. Any ideas for how we can better name these two classes to distinguish their purposes?

Reply:
```ParametersFileWriter```` is for vscode action feature and ```ParamsFileWriter``` is for our project. I was thinking that we could refer to vscode action feature one as ```ParametersPlaceholderFileWriter``` and the other one as just ```ParametersFileWriter```? Any other suggestions? @jeskew

Reply:
Renaming the existing `ParametersFileWriter` class seems reasonable to me. I had the same thought @anthony-c-martin mentions since it's not clear from the name that the existing class will generate a stub file based on a template.